drupal/modules/avoindata-explore/templates/avoindata_explore_block.html.twig

```diff
@@ -7,89 +7,98 @@
-          <div class="row container-fluid">
+        <div class="avoindata-explore-card-content">
+          <div class="avoindata-explore-card-title">
             <h3>
               {% trans %}
                 Datasets
               {% endtrans %}
             </h3>
           </div>
-          <div class="row container-fluid d-flex align-items-center">
+
+          <div class="avoindata-explore-card-text">
             <i class="far fa-chevron-right"></i> 
             <a href="/data/{{ language }}/dataset">
               {% trans %}
                Browse datasets
               {% endtrans %}
-               </a>
+            </a>
           </div>
         </div>
+
       </div>
 
-      <div class="row avoindata-explore-card" container-fluid d-flex>
-        <div class="col-md-4 align-items-center d-flex avoindata-explore-card-icon apiset">
+      <div class="avoindata-explore-card mt-2">
+        <div class="avoindata-explore-card-icon avoindata-explore-apiset">
           <img src="/themes/avoindata/images/exchange.svg"/>
         </div>
 
-        <div class="col-md-8 avoindata-explore-card-content pl-3">
-          <div class="row container-fluid">
+        <div class="avoindata-explore-card-content">
+          <div class="avoindata-explore-card-title">
             <h3>
               {% trans %}
                 APIs
               {% endtrans %}
             </h3>
           </div>
-          <div class="row container-fluid d-flex align-items-center">
+
+          <div class="avoindata-explore-card-text">
             <i class="far fa-chevron-right"></i> 
-            <a href="/data/{{ language }}/apiset">
+            <a href="/data/{{ language }}/dataset">
               {% trans %}
-                Browse apis
+               Browse apis
               {% endtrans %}
             </a>
           </div>
         </div>
+
       </div>
 
-      <div class="row avoindata-explore-card" container-fluid d-flex>
-        <div class="col-md-4 align-items-center d-flex avoindata-explore-card-icon showcase">
+      <div class="avoindata-explore-card mt-2">
+        <div class="avoindata-explore-card-icon avoindata-explore-showcase">
           <img src="/themes/avoindata/images/meter.svg"/>
         </div>
 
-        <div class="col-md-8 avoindata-explore-card-content pl-3">
-          <div class="row container-fluid">
+        <div class="avoindata-explore-card-content">
+          <div class="avoindata-explore-card-title">
             <h3>
               {% trans %}
                 Apps
               {% endtrans %}
             </h3>
           </div>
-          <div class="row container-fluid d-flex align-items-center">
-            <i class="far fa-chevron-right"></i> <a href="/data/{{ language }}/showcase">
+
+          <div class="avoindata-explore-card-text">
+            <i class="far fa-chevron-right"></i> 
+            <a href="/data/{{ language }}/dataset">
```

Comment:
Is this link updated from showcase => dataset for a purpose?

Reply:
Fixed in [a6d7b26](https://github.com/vrk-kpa/opendata/pull/1730/commits/a6d7b268a8685aa016145c4124821dccf0eecd1f)